### ISSUE TITLE
Generated site get stuck for serveral seconds every time when refreshing

### ISSUE THREAD

**Issue by wangx1ng** [mkdocs/mkdocs#915]
Mkdocs just generated a documentation site with 1000+ pages for me, everything is going well but any time when I click into any page, it freezes for several seconds and I can't do anything with my mouse, and then it goes back to normal.

First I thought maybe it's because there are too many sub tabs to show in my navigation bar, it spends too much time, so I hacked a little with my theme's nav.html so that only a little sub tabs (less than 40) will show. I tested with `mkdocs serve` again and it's the same thing, every time you go to a new page or refresh the current one, it get stuck for several seconds.

I have no idea why, is it because there are too many pages? But it shouldn't make any sense, there are static html files after all. Any help that takes me to the right direction of this would be appreciated.

System Environment: OS X 10.11
MkDocs Version: the current version which is 0.15.3
Theme: mkdocs-bootswatch: Journal
Python Interpreter: Python 3.4.x

**Comment by waylan** [comment]
This is almost certainly a duplicate of #859. We have a proposed fix, but it is currently incomplete and/or not fully tested. Any additional feedback would be appreciated.

**Comment by wangx1ng** [comment]
I deployed the site anyway and I found that it looks okay when visiting the site in a mobile browser, it won't freeze and it will response as soon as you touch any link on your page. I looked through #859, I'm just a user and don't know much about how browsers process web pages, so according what I see, I guess maybe it's because mobile browsers do something different from desktop browsers when processing javascript files? MkDocs is awesome! thank you so much for making it and I really wish this issue would be fixed soon.
